File: python/prism_sdk/throughput_audit_integrity_inference.py
"""Sweep P32 throughput inference audit-integrity feature F03."""
from .audit_integrity_support import AuditRequest4,AuditCard7,AuditIntegrityError,manifest,qualify
FEATURE_ID="AFA-sweep-P32-F03";CONTRACT_VERSION="sweep-throughput_audit_integrity_inference/1.0"
def throughput_audit_integrity_inference_manifest():return manifest(feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="throughput",mode="inference")
def qualify_throughput_audit_integrity_inference(request:AuditRequest4)->AuditCard7:return qualify(request,feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="throughput",mode="inference")

File: crates/sweep/src/throughput_audit_integrity_inference.rs
//! Sweep P32 throughput inference audit-integrity feature.
use super::audit_integrity_support::{manifest,qualify,AuditCard7,AuditIntegrityError,AuditRequest4};
pub const FEATURE_ID:&str="AFA-sweep-P32-F03";pub const CONTRACT_VERSION:&str="sweep-throughput_audit_integrity_inference/1.0";
pub fn throughput_audit_integrity_inference_manifest()->serde_json::Value{manifest(FEATURE_ID,CONTRACT_VERSION,"throughput","inference")}
pub fn qualify_throughput_audit_integrity_inference(request:&AuditRequest4)->Result<AuditCard7,AuditIntegrityError>{qualify(request,FEATURE_ID,CONTRACT_VERSION,"throughput","inference")}
